release-tools/util.sh

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env bash
+
+# Copyright 2014 The Kubernetes Authors.
```

Comment:
nit: should be 2019

Reply:
The original code actually is from 2014. I just copied it.